lib/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+
+app.use(cors());
+app.use(express.json());
+
+app.use('/api/v1/memer', require('./routes/memes'));
```

Comment:
'/api/v1/memes' like the LAB.md noted would make your heroku deploy work with Ryan's front end.